model/flow/identity.go

```diff
@@ -82,10 +96,15 @@ func ParseIdentity(identity string) (*Identity, error) {
 
 	// create the identity
 	iy := Identity{
-		NodeID:  nodeID,
-		Address: address,
-		Role:    role,
-		Weight:  weight,
+		IdentitySkeleton: IdentitySkeleton{
+			NodeID:        nodeID,
+			Address:       address,
```

Comment:
I think it is worth-while to keep the option open for node operators to change the address of their node. Though, in my mind, the address should still go into the `IdentitySkeleton`, because:
* The `IdentitySkeleton` contains the portions of the identity that are invariant throughout an epoch. If the address can change at epoch boundaries, wouldn't that be still compatible with the field being in `IdentitySkeleton`?
* In contrast `DynamicIdentity` holds the fields that can vary throughout an epoch. Not sure if we want to allow operators to change their address at arbitrary points in time.

Reply:
> If the address can change at epoch boundaries, wouldn't that be still compatible with the field being in IdentitySkeleton?

I agree we don't want to allow changing the network address at any point in time. I guess the fact that `IdentitySkeleton` fields are "invariant within an epoch" as opposed to just "invariant" hadn't clicked when I wrote the comment. Agree with your conclusion 👍